docs/how-to-guides/shared-libraries.rst

```diff
@@ -64,11 +64,16 @@ A shared library produced by ``library()`` or ``shared_library()`` built like th
 
 is installed to ``libdir`` by default. If the only reason the shared library exists
 is to be used inside the Python package being built, then it is best to modify
-the install location to be within the Python package itself:
+the install location, via the `install_dir` argument, to be within the Python package itself:
```

Comment:
reST requires double backticks to format something as code. Can you also line wrap this at 78-80 characters?